### PR TITLE
(maint) Minor string fixes from translator review

### PR DESCRIPTION
**Pull request by Magisus**

### CODE REVIEW

**Comment by LarissaLane** [comment]
@Magisus Is "Could not" externalized as a separate string somewhere? If so, translators won't be able to translate this into Japanese because "could not" needs to come at the end of the sentence in Japanese, not the beginning.

**Comment by Magisus** [comment]
It's externalized here: https://github.com/puppetlabs/puppet/blob/3e66932aa2d2d822626e35a46f04ce9922c086cf/lib/puppet/util.rb#L678

Not really sure what we can do about that...

**Comment by Magisus** [comment]
Updated to no longer split the `exit_on_fail` string.

File: lib/puppet/type/mount.rb [hunk 1]
```diff
@@ -65,7 +65,7 @@ module Puppet
                      # the wrong attributes so I sync AFTER the umount
           return :mount_unmounted
         else
-          raise Puppet::Error, _("Unexpected change from %{current} to unmounted}") % { current: current_value }
+          raise Puppet::Error, _("Unexpected change from %{current} to unmounted") % { current: current_value }
```
Comment (ahpook) [inline lib/puppet/type/mount.rb]: Good catch!

**Comment by ahpook** [comment]
Reviewed, LGTM!

**Pull request merged**